### ISSUE TITLE
Register package in General

### ISSUE THREAD

**Issue by FriesischScott** [FriesischScott/UncertaintyQuantification.jl#7]
I will register the package in the General repository of Julia to reserve the name.

**Comment by AnderGray** [comment]
Sound great!

We should also think of how much to add before we release 1.0.

Bayesian updating and copulas from my side is something I'd be interested in working on

**Comment by FriesischScott** [comment]
According to the Julia documentation before releasing `1.0.0` anything goes. We have to release a version to register the package, so I would release `0.1.0` with what we have now and then we can slowly add things to it.

**Comment by AnderGray** [comment]
Ok that's fine, although it's pretty basic for now, the line sampling is nice

**Comment by FriesischScott** [comment]
Yeah, it's a bit useless like this but at least it will be out there.

**Comment by FriesischScott** [comment]
@AnderGray I vaguely recall discussing this in Glasgow. Do you happen to remember what we ended up with regarding the license etc?

**Comment by AnderGray** [comment]
Personally I don't mind, do we need to use the same licence of cossan at all? I don't think we're using much of it's code

**Comment by FriesischScott** [comment]
Here we go: https://github.com/JuliaRegistries/General/pull/19667

**Comment by FriesischScott** [comment]
> Your new package pull request met all of the guidelines for auto-merging and is scheduled to be merged when the mandatory waiting period (3 days) has elapsed.

:partying_face:
